tracer/src/Datadog.Trace/LifetimeManager.cs

```diff
@@ -168,5 +229,99 @@ static void SetSynchronizationContext(SynchronizationContext? context)
                 }
             }
         }
+
+#if NET6_0_OR_GREATER
+        private static void TryLogWarning(Exception ex, string message)
+        {
+            try
+            {
+                // The message template must be a constant to avoid allocations and to satisfy logging analyzers.
```

Comment:
We don't need this comment here (or, for consistency, we should add it _everywhere_ we use log message templates 😅 )